src/vega.schema.ts

```diff
@@ -739,6 +761,8 @@ export interface VgAxisConfig extends VgAxisBase {
   titleY?: number;
 }
 
+export interface VgAxisConfig extends VgAxisBase {}
```

Comment:
If all properties of `VgAxisBase` are in `VgAxisConfig`, then we don't need to declare a separate `VgAxisBase`.

Reply:
I think `Axis` in `src/axis.ts` also extends `VgAxisBase` but has additional properties as well. Should I change `VgAxisBase` to `VgAxisConfig` and have `Axis` extend that?

Reply:
Yep